052020 (#2995)
biocore/qiita: CHANGELOG.md

# Qiita changelog

Version 052020
--------------

* Added Metatrascriptomicas as a data type, added a Ribosomal read filtering step and documentation on how to use it in the processing recommendations
* Fixed issue that prevented creating new artifacts when it was the children of a public parent

> * Qiita now keeps track of the delete artifact jobs which avoids creating duplicated delete jobs and the user gets this feedback in the GUI

Reviewer: ```suggestion
* Qiita now keeps track of artifact deletion jobs, prevents submitting duplicated deletions, and the GUI is updated when an artifact is being deleted
```

* We now display the `redbiom` DB release date in the redbiom Qiita page
* Fixed EBA-ENA duplicated sample submission in multiple preparations - this could happen when a sample existed in more than one preparation

> * Now is possible to deprecate a full preparation; this is useful when there is an unsuccessful run/preparation

Reviewer: ```suggestion
* Add the ability to deprecate a preparation; this is useful when there is an unsuccessful run or preparation
```

* The study page now has a markdown `Notes` section so users add problematic samples, explaining certain metadata columns, etc
* Added user documentation to better explain how to split your samples in preparations

> * Removed unused-legacy code to deal with duplicated sample names while building analyses

Reviewer: ```suggestion
* Fix a bug where repeated sample names were incorrectly handled during meta-analysis (#2978). Removed unused-legacy code to deal with duplicated sample names while building analyses.
```

* Improved headings in the stats page to avoid confusions

> * Fixed annoying issue that only deleted selected samples within the page viewing frame

Reviewer: ```suggestion
* Fixed issue that only deleted selected samples within the page's viewing frame
```

* Qiita now uses `gnu=True` in the `humanize.naturalsize` so the size display matches what a user sees when they `ls` the file

> * Updated code so it works with the latest version of `pandas`

Reviewer: Would you mind listing the exact version?
